src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -681,7 +681,7 @@ private static boolean shouldRemoveFinalVariableCandidate(ScopeData scopeData, D
      *         of the current ast node, null if no such node exists
      */
     private static DetailAST getLoopAstParent(DetailAST ast) {
```

Comment:
`getLoopAstParent` -> `getParentLoop`

Please do me a favor to rename method.

Reply:
Done, in addition to that i also renamed `loopAstParent` -> `parentLoop`